content/200-concepts/200-database-connectors/07-mongodb.mdx

```diff
@@ -3,17 +3,184 @@ title: 'MongoDB'
+| `tlsAllowInvalidCertificates`   |         | Bypasses the validation of the certificates used on the mongod/mongos instance. New in MongoDB version 4.2.                                                                                                                                                                                                                    |
+| `tlsAllowInvalidHostnames`      |         | Disables the hostname validation of the certificate used by the mongod/mongos instance. New in MongoDB version 4.2.                                                                                                                                                                                                            |
+| `tlsInsecure`                   |         | Disables certificate validations. New in MongoDB version 4.2.                                                                                                                                                                                                                                                                  |
+| `connectTimeoutMS`              |         | Time in milliseconds the before the connection times out.                                                                                                                                                                                                                                                                      |
+| `socketTimeoutMS`               |         | Time in milliseconds to send or receive on an open socket.                                                                                                                                                                                                                                                                     |
+| `socket_timeout`                |         | Maximum number of seconds to wait until a single query terminates.                                                                                                                                                                                                                                                             |
+| `maxPoolSize`                   | 100     | The maximum number of connections in the connection pool.                                                                                                                                                                                                                                                                      |
+| `minPoolSize`                   | 0       | The minimum number of connections in the connection pool.                                                                                                                                                                                                                                                                      |
```

Comment:
Is this our normal connection pool or something else? Does the logic generally work the same as for SQL databases?

Reply:
No :( It's the MongoDB driver's pool - it's in the QE, but configure differently